vertical-pod-autoscaler/hack/deploy-for-e2e.sh

```diff
@@ -44,7 +43,7 @@ fi
 SUITE=$1
 
 case ${SUITE} in
-  recommender|updater|admission-controller)
+  recommender|admission-controller)
```

Comment:
While I totally agree with removing the e2e test suite, here it might be useful to have a script that just lets you deploy updater. I know it's called deploy-for-e2e, but it is useful just for ad-hock testing of components while changing them,

Reply:
Good idea.
Changes reverted.